tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+from utils import produce_jwt
+from Crypto.PublicKey import RSA
+import os
+import jwt
+
+
+class Test_utils:
+
+    payload = None
+
+    def set_token(self):
+        if not self.payload:
+            with open(os.getenv('swag_pem'), "r") as f:
+                private_pem = f.read()
+            publickey = RSA.importKey(private_pem).publickey().exportKey('PEM')
+            jwtoken = produce_jwt()
+            self.payload = jwt.decode(jwtoken, publickey, algorithms='RS256')
+        return self.payload
+
+    def test_jwt_time(self):

```

Comment:
Your test names should be descriptive because they're only shown when they fail. Additionally, they should have docstrings explaining what they test. To get a better idea, check out the tests on the main repo https://github.com/SwagLyrics/SwagLyrics-For-Spotify